### ISSUE TITLE
issue with front page of Zim on a USB stick loading 

### ISSUE THREAD

**Issue by realeatham** [kiwix/kiwix-android#3809]
**Describe the bug**
See video

**Steps to reproduce the behavior:**
1. Connect USB with Zim
2. Load Zim
3. See a blank screen 

**Screenshots**
[Screen_Recording_20240428_221645_Kiwix.zip](https://github.com/kiwix/kiwix-android/files/15142174/Screen_Recording_20240428_221645_Kiwix.zip)
Sorry about it being in a zip, had to halve the file size.

**Environment**
- Version of Kiwix Android : 3.10.0
- Device : Samsung Galaxy S21 - SM-G991B 
- OS version : OneUI 6.0 Android 14 July 1 2023 Google Play system update Android security patch level April 1 2024

**Extra**
See #306 

@kelson42

**Comment by MohitMaliFtechiz** [comment]
@realeatham Thanks for providing the video. Before starting to investigate can you please provide a bit more information?

> The iFixit (internal storage) front page loads fine, so it is probably a issue with connecting a USB https://github.com/kiwix/kiwix-android/issues/306#issuecomment-2081455495

1. Have you tried to load the same zim file(Wikipedia) from internal storage?
2. Can you please try with [this small zim](https://download.kiwix.org/zim/other/alpinelinux_en_all_maxi_2023-01.zim) file on your USB stick and try to load the zim file? Is the behavior is same?
3. After loading the `Wikipedia` It shows a blank page. Can you please click on the `Home` icon(in the bottom bar) and see if its loads the main page or not?
![mainpage](https://github.com/kiwix/kiwix-android/assets/34593983/7beba5dd-b8cc-4a50-a4ae-7c14a8425969)

**Comment by realeatham** [comment]
1. No, and I won't as it is a large file and copying it takes ages.
2. Works 
3. Already done, loads blank page. The front page loads on the desktop pwa. Searching for the front page on the android version also returns a blank page.

**Comment by realeatham** [comment]
Just the app can't display it

**Comment by MohitMaliFtechiz** [comment]
@realeatham Thanks for your testing and quick response. We will investigate this issue.

**Comment by MohitMaliFtechiz** [comment]
@realeatham I have seen the same issue with my Samsung device(With my other devices front page is loading) "Front page" was showing a blank screen as shown in your video(This Samsung device used the old Chrome version). But after upgrading the Chrome browser it started showing the "front page" instead of a blank screen. 
Can you please upgrade your Chrome and try again(After upgrading the Chrome it took a few minutes to properly show the front page, probably it was updating the webView dependencies in the background). 

One additional step if upgrading the Chrome will not work for you.
1. Go to the application setting.
2. Go to `Apps` where all the installed apps list shown.
3. Find the `Android System WebView` and click on that.
4. Scroll down to the bottom, Click on `App details` it will take you to the playstore.
5. Update the webview if any update available there.

**Comment by realeatham** [comment]
I do not have chrome installed, and won't install it. There are no updates available for Android system webview and Firefox doesn't show up as an option in the webview selector.

**Comment by MohitMaliFtechiz** [comment]
@realeatham Thanks for providing the logs. Can you please try this apk and give your feedback, Does the problem still exist at your end https://drive.google.com/file/d/11JYfBfYf3H4dIUibZ0tuMlT_vtOEm0xn/view?usp=sharing?

**Comment by realeatham** [comment]
Unfortunately I cannot install the APK as it has been signed with a different signature to the main one and I cannot figure out how to export the zims downloaded from the app without another device

**Comment by realeatham** [comment]
Can you send a new one with a different app name so it can be installed side by side with the main one.

**Comment by kelson42** [comment]
@MohitMaliFtechiz I propose to just merge and release 3.10.0 like planned. If there is still a problem, then reopen the issue.
